deploy/awstools/aws_setup.py

```diff
@@ -0,0 +1,64 @@
+#/usr/bin/env python3
+
+""" This script configures your AWS account to run FireSim. """
```

Comment:
This used to live at `scripts/aws-setup.py`.  I moved it here because it's easier to import and use in tests for code that assume this has been run.  `scripts/aws-setup.py` still exists and just imports this and runs it.